src/homeblks/volume/volume.cpp

```diff
@@ -335,7 +335,96 @@ indx_tbl* Volume::recover_indx_tbl(btree_super_block& sb, btree_cp_sb& cp_info)
     return static_cast< indx_tbl* >(tbl);
 }
 
+static bool is_buf_zero(const uint8_t* buf, size_t size) {
```

Comment:
Can we move below functions to hs_utils since they are common and in case other layers could reuse this function is needed?

Reply:
done